build/SignToolData.json

```diff
@@ -4,15 +4,133 @@
             "certificate": "Microsoft402",
             "strongName": "MsSharedLib72",
             "values": [
-                "bin/Sdks/Microsoft.NET.Sdk/tools/netcoreapp1.0/Microsoft.NET.Build.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/netcoreapp1.0/*/Microsoft.NET.Build.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/net46/Microsoft.NET.Build.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/net46/*/Microsoft.NET.Build.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/netcoreapp1.0/Microsoft.NET.Build.Extensions.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/netcoreapp1.0/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/net46/Microsoft.NET.Build.Extensions.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/net46/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll",
+                "bin/Sdks/Microsoft.NET.Sdk/tools/*/Microsoft.NET.Build.Tasks.dll",
+                "bin/Sdks/Microsoft.NET.Sdk/tools/*/*/Microsoft.NET.Build.Tasks.resources.dll",
+                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/*/Microsoft.NET.Build.Extensions.Tasks.dll",
+                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/*/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll"
+            ]
+        },
+        {
+            "certificate": null,
+            "strongName": null,
+            "values": [
+                "packages/*.nupkg"
             ]
         }
+    ],
+    "exclude": [
```

Comment:
Do you really need this exclude section if you are being explicit on the dlls to sign above?

Reply:
Yes. The SignTool expects all binaries in containers (nuget, vsix, etc) to be explicitly signed or explicitly excluded.

Reply:
CC. @jaredpar